### ISSUE TITLE
[CAL-292] Displays the wrong availability

### ISSUE THREAD

**Issue by newz666** [calcom/cal.com#5271]
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Accessed a user's event link who's situated in PST. I am located in EST. It displayed the wrong availability

### Steps to Reproduce

1. Clicked on the event link of a user who is in Pacific Time
2. It was displayed the user is available starting at 11am EST (which is 8am PT), but the user availability start at 9am PT / 12 EST/

![](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/c7cb03da-ceda-4cf8-a264-3ab56525b936/24726038-d1bf-4d8f-b732-d528a13cd649)

![](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/de81725b-ee4c-4085-8533-b7d071c798e2/69e57628-bc9f-4bbc-bfc8-019ff4ccf119)

![](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/82919ca3-5f6c-4344-88a3-7007e1a1f69b/79035b69-0366-4973-91f2-23018f6bea04)

### Technical details

Self Hosted, main branch v 2.1.1

> From [Linear-GitHub Sync](https://synclinear.com)

**Comment by PeerRich** [comment]
do you have a video?

**Comment by newz666** [comment]
@PeerRich please see the recording bellow. 

[Screencast from 10-30-2022 07:32:20 PM.webm](https://user-images.githubusercontent.com/74431325/198907553-ac9e3c98-9dcc-4e87-968c-2098b487952c.webm)

**Comment by melissamarvin** [comment]
1. set your own user to some timezone.  I set mine to CST
2. click settings
3. click impersonate under admin
4. choose a person in a different timezone.  I used Octavian B.  He was set to EST
5. click bookings
6. click one of the appointments
7. click reschedule

Notice the event has Pacific selected.  If you modify to the timezone the person is actually in, the time slots available don't modify themselves to the correct timezone availabilities.  


https://user-images.githubusercontent.com/117048096/199025001-f1d52eac-5e27-41ec-b011-c7de2c8a667e.mp4
